data/modules/PolicePatrol/PolicePatrol.lua

```diff
@@ -169,13 +163,37 @@ local onEnterSystem = function (player)
 			end)
 		end
 	end
+
+	local police = ShipDef[system.faction.policeShip]
+	Timer:CallEvery(15, function ()
+		if not Game.system or #patrol == 0 then return true end
+		if target then return end
```

Comment:
This ideally should be an explicit `return false` - the function is contractually bound to return a boolean value indicating whether the timer should be cancelled. While this does work currently, it's more confusing to future readers this way.